projects/ngx-signature-pad/src/lib/ngx-signature-pad.component.ts

```diff
@@ -234,7 +242,7 @@ export class NgxSignaturePadComponent implements OnInit, OnChanges {
 
   constructor(private renderer2: Renderer2, private overlay: Overlay, private viewContainerRef: ViewContainerRef) {}
 
-  ngOnInit(): void {
+  ngAfterViewInit(): void {
```

Comment:
Why use this life cycle instead of `ngOnInit`?

Reply:
because the element's @ViewChild only exists after it is initialized in the view. In ngOnInit the @ViewChild cannot recognize element as ElementRef